website/src/documentation/sdks/python-dependencies.md

```diff
@@ -26,460 +26,8 @@ behavior in the service. If you are using any of these packages in your code, be
-  <tr><td>google-cloud-pubsub</td><td>==0.26.0</td></tr>
-  <tr><td>google-cloud-bigquery</td><td>==0.25.0</td></tr>
-  <tr><td>googledatastore</td><td>==7.0.1</td></tr>
-  <tr><td>grpcio</td><td>&gt;=1.8, &lt;2</td></tr>
-  <tr><td>hdfs</td><td>&gt;=2.1.0, &lt;3.0.0</td></tr>
-  <tr><td>httplib2</td><td>&gt;=0.8, &lt;=0.11.3</td></tr>
-  <tr><td>mock</td><td>&gt;=1.0.1, &lt;3.0.0</td></tr>
-  <tr><td>nose</td><td>&gt;=1.3.7</td></tr>
-  <tr><td>numpy</td><td>&gt;=1.14.3, &lt;2</td></tr>
-  <tr><td>oauth2client</td><td>&gt;=2.0.1, &lt;5</td></tr>
-  <tr><td>proto-google-cloud-datastore-v1</td><td>&gt;=0.90.0, &lt;0.90.4</td></tr>
-  <tr><td>proto-google-cloud-pubsub-v1</td><td>==0.15.4</td></tr>
-  <tr><td>protobuf</td><td>&gt;=3.5.0.post1, &lt;4</td></tr>
-  <tr><td>pydot</td><td>&gt;=1.2.0, &lt;1.3</td></tr>
-  <tr><td>pyhamcrest</td><td>&gt;=1.9, &lt;2.0</td></tr>
-  <tr><td>pytz</td><td>&gt;=2018.3, &lt;=2018.4</td></tr>
-  <tr><td>pyyaml</td><td>&gt;=3.12, &lt;4.0.0</td></tr>
-  <tr><td>pyvcf</td><td>&gt;=0.6.8, &lt;0.7.0</td></tr>
-  <tr><td>six</td><td>&gt;=1.9, &lt;1.12</td></tr>
-  <tr><td>typing</td><td>&gt;=3.6.0, &lt;3.7.0</td></tr>
-</table>
-
-</details>
-
-<details><summary markdown="span"><b>2.6.0</b></summary>
-
-<p>Beam SDK for Python 2.6.0 has the following compile and
-  runtime dependencies.</p>
-
-<table class="table-bordered table-striped">
-  <tr><th>Package</th><th>Version</th></tr>
-  <tr><td>avro</td><td>&gt;=1.8.1,&lt;2.0.0</td></tr>
-  <tr><td>crcmod</td><td>&gt;=1.7,&lt;2.0</td></tr>
-  <tr><td>dill</td><td>&gt;=0.2.6,&lt;=0.2.8.2</td></tr>
-  <tr><td>future</td><td>&gt;=0.16.0,&lt;1.0.0</td></tr>
-  <tr><td>futures</td><td>&gt;=3.1.1,&lt;4.0.0</td></tr>
-  <tr><td>google-apitools</td><td>&gt;=0.5.18,&lt;=0.5.20</td></tr>
-  <tr><td>google-cloud-bigquery</td><td>==0.25.0</td></tr>
-  <tr><td>google-cloud-pubsub</td><td>==0.26.0</td></tr>
-  <tr><td>googledatastore</td><td>==7.0.1</td></tr>
-  <tr><td>grpcio</td><td>&gt;=1.8,&lt;2</td></tr>
-  <tr><td>hdfs</td><td>&gt;=2.1.0,&lt;3.0.0</td></tr>
-  <tr><td>httplib2</td><td>&gt;=0.8,&lt;=0.11.3</td></tr>
-  <tr><td>mock</td><td>&gt;=1.0.1,&lt;3.0.0</td></tr>
-  <tr><td>oauth2client</td><td>&gt;=2.0.1,&lt;5</td></tr>
-  <tr><td>proto-google-cloud-datastore-v1</td><td>&gt;=0.90.0,&lt;=0.90.4</td></tr>
-  <tr><td>proto-google-cloud-pubsub-v1</td><td>==0.15.4</td></tr>
-  <tr><td>protobuf</td><td>&gt;=3.5.0.post1,&lt;4</td></tr>
-  <tr><td>pydot</td><td>&gt;=1.2.0,&lt;1.3</td></tr>
-  <tr><td>pytz</td><td>&gt;=2018.3,&lt;=2018.4</td></tr>
-  <tr><td>pyvcf</td><td>&gt;=0.6.8,&lt;0.7.0</td></tr>
-  <tr><td>pyyaml</td><td>&gt;=3.12,&lt;4.0.0</td></tr>
-  <tr><td>six</td><td>&gt;=1.9,&lt;1.12</td></tr>
-  <tr><td>typing</td><td>&gt;=3.6.0,&lt;3.7.0</td></tr>
-</table>
-
-</details>
+You can also retrieve a dependency list by creating a clean virtual environment for your project, installing the Beam SDK for Python, and running `pip3 freeze` or `pip freeze`.
```

Comment:
I kind of feel like we should give the reader the full instructions here to mitigate the docs regression as much as possible. I think something like this should be enough, if you agree...

1. If the `virtualenv` package is not installed, run `pip3 install virtualenv` or `pip install virtualenv`
2. `git clone https://github.com/apache/beam.git`
3. `virtualenv py_deps && source py_deps/bin/activate`
4. `pip3 freeze` or `pip freeze`

Reply:
- This list will include transitive dependencies. It is easier to make sense of these via `pip install pipdeptree; pipdeptree -p apache-beam`

- ip command in virtualenv works under Python3 as well. 

- I would not duplicate installation instructions here, but point to webpage that talks on how to install beam SDK. For example, it is possible to install beam from pypi, instead of cloning.

Reply:
Added in explicit steps, but wasn't sure on our recommended format for breaking out Python 3 v. Python 2 instructions. If my update goes against some established style, please let me know.